src/HealthChecks/HealthChecks/src/Microsoft.Extensions.Diagnostics.HealthChecks.csproj

```diff
@@ -12,6 +12,8 @@ Microsoft.Extensions.Diagnostics.HealthChecks.IHealthChecksBuilder
     <PackageTags>diagnostics;healthchecks</PackageTags>
     <IsAspNetCoreApp>true</IsAspNetCoreApp>
     <Nullable>enable</Nullable>
+    <!-- workaround https://github.com/dotnet/roslyn/issues/52527 -->
+    <NoWarn>$(NoWarn);SYSLIB1006</NoWarn>
```

Comment:
Undo?

Reply:
Even though the underlying roslyn issue is closed, I'm still seeing the warnings in VS without this change

![image](https://user-images.githubusercontent.com/8291187/124010227-2e533100-d9a4-11eb-98ad-16886007c265.png)

Reply:
The reason this is still needed is because we are using an older Roslyn here:

https://github.com/dotnet/aspnetcore/blob/b9efadc59ddbf6743775244622fb95301dd1846c/eng/Versions.props#L161-L162

That version didn't have the fix for https://github.com/dotnet/roslyn/issues/52527